source/OutfitterPanel.cpp

```diff
@@ -385,14 +385,14 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 		int64_t credits = player.Accounts().Credits();
 
 		if(cost > credits)
-			errors.push_back("You do not have enough money to buy this outfit, you need a further " +
+			errors.push_back("You do not have enough money to buy this outfit. You need a further " +
```

Comment:
Semicolon could also work here.

Reply:
I re-broke this in #10743, and have just fixed it locally.